### PR TITLE
[MOBILE-627] add more logging

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Adds trace logging to the primary (non-helper) method calls, listener setting and event firing and adds debug logging to takeOff and plugin initialization.
 
### Why are these changes necessary?
To make the plugin easier to debug.

### How did you verify these changes?
Built/ran cordova sample

### CODE REVIEW

File: src/ios/UAirshipPlugin.m [hunk 1]
```diff
@@ -211,15 +228,18 @@ - (void)setUserNotificationsEnabled:(CDVInvokedUrlCommand *)command {
 }
 
 - (void)setLocationEnabled:(CDVInvokedUrlCommand *)command {
+    UA_LTRACE("setLocationEnabled called with command arguments: %@", command.arguments);
+
     [self performCallbackWithCommand:command withBlock:^(NSArray *args, UACordovaCompletionHandler completionHandler) {
         BOOL enabled = [[args objectAtIndex:0] boolValue];
-        [UAirship location].locationUpdatesEnabled = enabled;
```
Comment (BrianBatchelder) [inline src/ios/UAirshipPlugin.m]: Is this change intended?

Comment (crow) [reply src/ios/UAirshipPlugin.m]: no it snuck in from me breaking these out

**Comment by crow** [comment]
force push was just me removing commits I accidentally pushed to this branch instead of the parent
